manifests/hub/hub-clusterrole.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:open-cluster-management:{{ .HubCoreName }}
+rules:
+# Allow hub to monitor and update status of csr
+- apiGroups: ["certificates.k8s.io"]
+  resources: ["certificatesigningrequests"]
+  verbs: ["create", "get", "list", "watch"]
+- apiGroups: ["certificates.k8s.io"]
+  resources: ["certificatesigningrequests/status"]
```

Comment:
I suspect the operator will end up needing approval power, but I guess you'll find out.

Reply:
fixed